roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -268,138 +291,202 @@ def get_image_size(self):
         return self.image_masks.shape[:2]
 
     @staticmethod
-    def write_segmentation(segext_obj, savepath, metadata_dict, **kwargs):
-        if isinstance(metadata_dict, str):
-            with open(metadata_dict, 'r') as f:
-                metadata_dict = yaml.safe_load(f)
-
-        # NWBfile:
-        nwbfile_args = dict(identifier=str(uuid.uuid4()), )
-        if 'NWBFile' in metadata_dict:
-            nwbfile_args.update(**metadata_dict['NWBFile'])
-        nwbfile = NWBFile(**nwbfile_args)
-
-        # Subject:
-        if 'Subject' in metadata_dict:
-            nwbfile.subject = Subject(**metadata_dict['Subject'])
-
-        # Device:
-        if isinstance(metadata_dict['ophys']['Device'], list):
-            for devices in metadata_dict['ophys']['Device']:
-                nwbfile.create_device(**devices)
+    def get_nwb_metadata(sgmextractor):
```

Comment:
I don't understand the purpose of this method.

Reply:
@bendichter there is a separate PR for the NWB write method (https://github.com/catalystneuro/roiextractors/pull/20)

Should I just port the changes here? so it includes all write segmentation methods